### DISCUSSION TITLE
Secret scanning: Is it possible to non-publicly inform a user of an issue on PR?

### DISCUSSION THREAD

**Discussion by johlju** [community/community#64070] [Code Security] [answered]
### Select Topic Area

Question

### Body

If there is a detection in a public repository, is the user of an issue or PR notified as well as the repository owner? If not, it is possible to non-publicly inform a user that there have been a detection and they might need to take action?

**Comment by Murat-Karakaya** [comment, marked as answer]
Indeed, security advisories serve exactly that purpose. When you visit a GitHub repository, right above the sections for issues, pull requests, and settings, you'll find a button labeled "Security". Click it. Then on the left side, you'll see more buttons – go ahead and click on 'Advisories.' This is where you can have a private discussion about any security concerns.

**Reply by Murat-Karakaya** [reply]
I hope this helps. For more info, you can read github's document [here](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability)

**Comment by johlju** [comment]
It helped. Thank you.

**Reply by Murat-Karakaya** [reply]
I am glad it did :)
